collections/Makefile

```diff
@@ -5,8 +5,11 @@ all: build perf
 
 motoko:
 	cd motoko; \
+	envsubst < mops.template.toml > mops.toml; \
+	mops install; \
```

Comment:
Is this needed? Does `vessel source` call `vessel install`, but `mops source` doesn't do that? If so, we may need a ticket to mops.

Reply:
`mops sources` should install packages in the same way as Vessel (this was included a few months ago).